build.gradle.kts

```diff
@@ -409,12 +409,23 @@ tasks {
           "cody.autocomplete.enableFormatting" to
               (project.property("cody.autocomplete.enableFormatting") ?: "true"))
 
-  fun getIdeaInstallDir(ideaVersion: String, ideaType: String): File? {
+  fun getPlatformName(ideaType: String) =
```

Comment:
https://github.com/JetBrains/intellij-platform-gradle-plugin/blob/main/src/main/kotlin/org/jetbrains/intellij/platform/gradle/IntelliJPlatformType.kt

Reply:
Not yet available with 1.17.4, we must move to the new gradle plugin (v2).

Reply:
Maybe just after launch it will be good time to do that anyway?